sap_hana/datadog_checks/sap_hana/sap_hana.py

```diff
@@ -189,6 +194,21 @@ def query_database_status(self):
                 self.SERVICE_CHECK_STATUS, db_status, message=message, tags=tags, hostname=self.get_hana_hostname()
             )
 
+    def query_audit_logs(self):
+        # https://help.sap.com/docs/SAP_HANA_PLATFORM/4fe29514fd584807ac9f2a04f6754767/d1fe1244d29510148f69be8b0e060dcc.html
+        for audit_log in self.iter_rows(queries.AuditLog()):
+            data = exclude_undefined_keys(audit_log)
+
+            data['status'] = data.pop('event_level')
+            data['timestamp'] = get_timestamp(data['timestamp'])
+
+            message = '{} {}'.format(data['event_action'], data['event_status'])
+            if 'comment' in data:
+                message += ' | {}'.format(data['comment'])
+            data['message'] = message
+
+            self.send_log(data)
```

Comment:
There is timestamp information in the data we get. Was it a requirement that we don't do any checkpointing or is adding checkpointing going to be our first support case? ;)

Reply:
Done!

Reply:
💪 mega, thanks!